libs/entitlement/bridge/src/main/java/org/elasticsearch/entitlement/bridge/package-info.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+/**
+ * <p>
```

Comment:
nit: this opening paragraph tag isn't necessary. They just need to be between paragraphs. From Oracle's [docs](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html):
> If you have more than one paragraph in the doc comment, separate the paragraphs with a `<p>` paragraph tag, as shown.